.dockerignore

```diff
@@ -0,0 +1,2 @@
+tools/
+node_modules/
```

Comment:
the build process runs `npm ci` anyways, so there's no real point to copying over 50 MB of node modules on every build